scripts/preinstall.js

```diff
@@ -113,6 +116,23 @@ async function main() {
 
     process.exit(1);
   }
+
+  process.exit(0);
```

Comment:
This line won't be hit with the early `return` above

Reply:
I'm not even really sure this `process.exit()` is necessary, but I reverted to back to the way it was before.